documents/motions/10.2019.1 - Advisory Council.md

```diff
@@ -0,0 +1,29 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 10.2019.1  
+**Subject:** WCA Advisory Council  
+**Intent:** Rights and duties of the WCA Advisory Council  
+**Submitted by:** Board of Directors  
+**Date:** January 1, 2019  
+
+# Motion
+
+The WCA Advisory Council is an Advisory Committee of the WCA.
+
+1. The WCA Advisory Council is responsible for building and maintaining bilateral communication between the WCA Community and the WCA as an organization.
+2. The WCA Advisory Council has the following rights and duties:
+   1. Establishing communication channels between the WCA Community and the WCA Board, providing direct feedback from Registered Speedcubers about the WCA as an organization.
+      1. The WCA Advisory Council shall ensure that the demands and requests of the WCA Community are heard within the WCA organization and are addressed by the WCA Board or the appropriate Committee(s)/Team(s).
+   2. Recruiting Registered Speedcubers who are abel and willing to help improve the WCA organization.
+      1. The WCA Advisory Council shall provide a platform for Registered Speedcubers who are volunteering to help the WCA in any way they could.
+   3. Improving and further promoting transparency in the WCA.
+   4. Improving the communication within the WCA organization.
+3. The WCA Advisory Council shall have the following types of representatives:
```

Comment:
We haven't said anywhere what a representative is, we should introduce that before speaking of the types.